javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/IndyProxy.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.bootstrap;
+
+/** Interface added to indy proxies to allow unwrapping the proxy object */
+public interface IndyProxy {
```

Comment:
If we want to make this a public api then I think we need to find a better name for this. Using indy is just an implementation detail.

Reply:
Does something like `InstrumentationProxy` or `InjectedProxy` would better fit here ?

Reply:
yes, I think either of these would be better